components/search_bar/search_bar.jsx

```diff
@@ -81,15 +81,16 @@ export default class SearchBar extends React.PureComponent {
         }
     }
 
-    componentWillReceiveProps(nextProps) {
-        const {searchTerms} = nextProps;
+    static getDerivedState(props, state) {
```

Comment:
I think this needs to be `getDerivedStateFromProps`

Reply:
Ah, that was an huge mistake 😅.